ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__multi-nightly-4.19-upgrade-from-stable-4.19.yaml

```diff
@@ -282,6 +282,7 @@ tests:
       ADDITIONAL_ENABLED_CAPABILITIES: CloudCredential CloudControllerManager Ingress
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      ENABLE_OTA_TEST: OCP-73352
```

Comment:
Shall we use a y-stream job instead of z-stream job? `f60` seems too long to get a test result.

Reply:
replaced this one with 4.18 -> 4.19